templates/docs/examples/layouts/brochure-site/_25-25-50.html

```diff
@@ -1,16 +1,16 @@
 <section class="p-strip u-no-padding--top">
-  <hr class="is-fixed-width u-no-margin--bottom">
-  <div class="row">
-    <div class="col-3 col-medium-3">
+  <hr class="p-rule">
```

Comment:
`p-rule` is not part of this branch and doesn't do anything.

Reply:
So for this PR we should revert to previous `hr` with class names in all occurences of `p-rule`, and use it only it PR where rule component is introduced
```suggestion
  <hr class="is-fixed-width u-no-margin--bottom">
```